### DISCUSSION TITLE
How do I hide font requests for sandboxed programs? (SBIE2205)

### DISCUSSION THREAD

**Discussion by Aholicknight** [sandboxie-plus/Sandboxie#2776] [Documents and notes]
![SandMan_20230317_XodtZWLOPG](https://user-images.githubusercontent.com/7843719/229248997-a7bacc7d-6dbb-4376-af41-5b24140fa28e.png)

I keep getting this message whenever I play a game that's sandboxed. How can I fix this, so it doesn't flood me with messages?

**Comment by offhub** [comment]
Click the down arrow next to the `Dismiss` button to open the drop-down menu and select "`Hide all such messages`".

**Reply by isaak654** [reply]
It looks like this is classified as bug in #545

**Reply by Aholicknight** [reply]
I did this many times, even clicked "never show this message again" and it shows up.

**Reply by isaak654** [reply]
Update: https://github.com/sandboxie-plus/Sandboxie/issues/545#issuecomment-1516395730
